config/hooks/normal/0530-linuxcnc-ethercat-sources.hook.chroot

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+curl -s https://packagecloud.io/install/repositories/rodw-au/rodw-au/script.deb.sh | sudo bash
```

Comment:
Downloading random stuff from the Internet and running it as root seem like a very bad idea to me.  This could compromise the build host and any images generated at the whim of those controlling the result from the URL provided.

Reply:
I can change this to be a more normal format. It is a very cool script though!

Reply:
This has now been revised to a local script (hook 530)